src/webgpu/capability_info.ts

```diff
@@ -1048,6 +1048,19 @@ export const kLimitInfo = /* prettier-ignore */ makeTable(
 /** List of all entries of GPUSupportedLimits. */
 export const kLimits = keysOf(kLimitInfo);
 
+/** List of all GPUFeatureName values. */
+export const kFeatureNames: readonly GPUFeatureName[] = [
+  "depth-clip-control",
+  "depth32float-stencil8",
+  "texture-compression-bc",
+  "texture-compression-etc2",
+  "texture-compression-astc",
+  "timestamp-query",
+  "indirect-first-instance",
+  "shader-f16",
+  "bgra8unorm-storage",
+];
```

Comment:
Best to use the object pattern, like this:
https://github.com/gpuweb/cts/blob/63c716bd354b19616360b6e7047d1508d40e7c89/src/webgpu/capability_info.ts#L70-L78

This looks really silly, but ensures the list here stays up to date with the list in the type definitions (which is generated from the spec).

Reply:
Done.